content/blog/2020-11-25-november-20-community-gems.md

```diff
@@ -0,0 +1,195 @@
+---
+title: November '20 Community Gems
+date: 2020-11-25
+description: |
+  A roundup of technical Q&A's from the DVC community. This month, learn how
+  to clean your cache and use Git hooks with DVC. And here's an early holiday 
+  gift- new Bitbucket support for CML!
+descriptionLong: |
+  A roundup of technical Q&A's from the DVC community. This month, learn how
+  to clean your cache and use Git hooks with DVC. And here's an early holiday 
+  gift- new Bitbucket support for CML!
+picture: 2020-11-25/cover.png
+author: elle_obrien
+commentsUrl: https://discuss.dvc.org/t/october-20-community-gems/535
+tags:
+  - Discord
+  - Gems
+  - CML
+  - Cache
+  - Bitbucket
+---
+
+## DVC questions
+
+### [Q: If I make a new Git branch, how can I synchronize it with DVC?](https://discord.com/channels/485586884165107732/485596304961962003/773498570795778058)
```

Comment:
to be honest, question is a bit strange as well ... if it's a new branch - we don't need to synchronize anything, right?

Reply:
i had that thought too.... i thought maybe i was misunderstanding. checkout hook is more for when you switch to a branch that has a different workspace, which by definition would only happen when the branch isn't brand new. Maybe we can change question to

`If i checkout a different Git branch, how do I synchronize with DVC?` Not exactly what was in the channel but more useful, maybe

Reply:
> checkout hook is more for when you switch to a branch that has a different workspace, which by definition would only happen when the branch isn't brand new

yep!

> Not exactly what was in the channel but more useful, maybe

yep, sounds def better!